### ISSUE TITLE
Volunteers to Update UI

### ISSUE THREAD

**Issue by demion92** [almalence/OpenCamera#45]
I love the functionality of this camera application, but the UI could use some improvement. Why not have a good UI for a great application? Are there any devs out there who would like to help me update the UI?

**Comment by Smeet123** [comment]
Yes I can redesign the entire application with Material design. But what is the benefits for me ? What could you give me in returns ?

**Comment by demion92** [comment]
@Smeet123 i'm not giving you shit. i'll do it myself. i was simply asking if anyone was willing to help.

**Comment by AABatteries** [comment]
I could help a bit with the design, although I'm just a graphic designer, not a coder/dev unfortunately. If you need any sort of such help, I'll be happy to do so.

**Comment by demion92** [comment]
@AABatteries , that would be awesome as I'm a coder and not a designer lol. Do you want to schedule a time to speak?

**Comment by gdaunton** [comment]
@demion92 did you end up making any progress on this? I was thinking of making a fork to work on the UI since this repo doesn't seem like it is very well maintained.

**Comment by DeastinY** [comment]
I actually like tthe UI.

**Comment by farazhassan** [comment]
Guys, I'd love to do any UI work required, just get me a design and I'll get it done ASAP.
